### ISSUE TITLE
remap command for new tab (remap 't')

### ISSUE THREAD

**Issue by kahunaoplomo** [philc/vimium#3588]
Hi,

Not an issue per se, but I thought this is the best way to ask about this.

Is it possible to remap a command to open new tab from t to another key? I have not found this command described anywhere. In other extensions of this type I think command was called 'newTab', but it doesn't seem to work with vimium.

Thanks for your work on this amazing extension.
Tym

**Comment by gdh1995** [comment]
It's createTab.



---Original---

**Comment by kahunaoplomo** [comment]
Thank you, that works. Is it documented somewhere? I couldn't find it even within advanced commands.

**Comment by jmmzzei** [comment]
> Thank you, that works. Is it documented somewhere? I couldn't find it even within advanced commands.

Yes. Inside Vimium options, click on the link "Show available commands" placed next to your custom key mappings.
